<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicationWidget</class>
 <widget class="QWidget" name="CommunicationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommunicationWidget</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Command</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="boxCommand"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonSend">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Answer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="boxAnswer"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
